Week01/Problem02/nkovacs_02.py

```diff
@@ -0,0 +1,17 @@
+def fib(max_value=100):
+	fib_seq = [1]
+	new_term = 2
+	while new_term <= max_value:
+		fib_seq.append(new_term)
+		new_term += fib_seq[-2]
+
+	return(fib_seq)

```

Comment:
Constructing the whole sequence is fine here, as the list doesn't get very long, but if you don't really need to keep the full thing it will help reduce memory overhead.